docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -37,129 +37,223 @@ We call the cluster with the HyperShift operator installed as the hosting servic
-
-```bash
-$ oc create secret generic hypershift-operator-oidc-provider-s3-credentials --from-file=credentials=$HOME/.aws/credentials --from-literal=bucket=<s3-bucket-for-hypershift> --from-literal=region=<region> -n <managed-cluster-used-as-hosting-service-cluster>
-```
-
-Add the special label to the `hypershift-operator-oidc-provider-s3-credentials` secret so that the secret is backed up for disaster recovery.
-
-```bash
-$ oc label secret hypershift-operator-oidc-provider-s3-credentials -n <managed-cluster-used-as-hosting-service-cluster> cluster.open-cluster-management.io/backup=true
-```
-
+1. If you plan to provision hosted clusters on the AWS platform, create an OIDC s3 credential secret for the HyperShift operator, and name it `hypershift-operator-oidc-provider-s3-credentials`. It should reside in managed cluster namespace (i.e., the namespace of the managed cluster that will be used as the hosting service cluster). If you used `local-cluster`, then create the secret in the `local-cluster` namespace
+
+    The secret must contain 3 fields:
+
+    - `bucket`: An S3 bucket with public access to host OIDC discovery documents for your HyperShift clusters
+    - `credentials`: Credentials to access the bucket
+    - `region`: Region of the S3 bucket
+
+    For details, please check: [HyperShift Project Documentation](https://hypershift-docs.netlify.app/getting-started). For convenience, you can create this secret using the CLI by:
+
+    ```bash
+    $ oc create secret generic hypershift-operator-oidc-provider-s3-credentials --from-file=credentials=$HOME/.aws/credentials --from-literal=bucket=<s3-bucket-for-hypershift> --from-literal=region=<region> -n <managed-cluster-used-as-hosting-service-cluster>
+    ```
+
+    Add the special label to the `hypershift-operator-oidc-provider-s3-credentials` secret so that the secret is backed up for disaster recovery.
+
+    ```bash
+    $ oc label secret hypershift-operator-oidc-provider-s3-credentials -n <managed-cluster-used-as-hosting-service-cluster> cluster.open-cluster-management.io/backup=true
+    ```
+
+    ### AWS Private Link
+    If you plan to provision hosted clusters on the AWS platform with Private Link, create an AWS credential secret for the HyperShift operator, and name it `hypershift-operator-private-link-credentials`. It should reside in managed cluster namespace (i.e., the namespace of the managed cluster that will be used as the hosting service cluster). If you used `local-cluster`, then create the secret in the `local-cluster` namespace
+
+    Follow the instructions here (steps 1-5): [Deploy AWS private clusters](https://hypershift-docs.netlify.app/how-to/aws/deploy-aws-private-clusters/)
+
+    Instructions result in the following AWS resources:
+
+    * IAM policy in AWS
+    * Hypershift operator IAM user
+    * Attach the policy to the user
+    * Create an IAM access key for the user, use the access key and secret in the next section
```

Comment:
", use" , maybe add "and" or put the "user ...." in parenthesis?

Reply:
Reworded